_posts/2019-10-24-Standing-Desk.md

```diff
@@ -0,0 +1,13 @@
+# Standing Desk
+
+We have a standing desk at the office. It is usually very lonely. I can count on two hands the number of times people apart from me and a coworker have used it in the last 3 months. I can also count on one hand the number of people who have used it and we have 40+ people in the office.
+
+Sitting for 8 hours a day, every day, is really bad for your health. It is usually even more than that since most people (me included) sit a lot when they get home as well.
+
+It used to be something that most people didn't hear about, but now everybody knows it. The difference is you can choose to act on it or not.
+
+The first step is to get up and take a stroll every once in a while. In doesn't need to be long, just a few minutes will do it. You will still be sitting most of the times, but it is going to be better than sitting **all the times**. I'm the furthest away from the water dispenser so I have a good excuse to get up and walk every few hours.
+
+I'm now trying to use the standing desk at least 2 hours every other day. It is not a lot, but it is a good starter.
+
+Just try one with no commitment. In the beginning I thought that I would be less productive, but it is not true at all. You just need to get used to it.
```

Comment:
What about the one that moves? It's the same one?

And you never gave an update on pomodoro...